docs/netserver/config/webservices.md

```diff
@@ -21,9 +21,14 @@ Configuration section used to define SuperOffice CRM web service settings.
 </WebServices>
 ```
 
-| Name | Description |
-|---|---|
-| AllowWebServiceRequests | Permit requests to web services. This value is supported in the default `SoWcfRequestInterceptor` and can be overridden by making a custom `SoWcfRequestInterceptorPlugin`. |
-| WrapExceptions | Check if NetServer shall wrap and serialize exceptions, or leave it up to the communication carrier (WCF). |
-| RemoveInvalidXMLText | If we should remove UTF-8 characters that are not valid XML. The Text table is the only place where this filter is active. |
-| RemoteBaseURL | Base URL for remote services. Will overridden by `Services.RemoteBaseURL`.<br>Default: `http://localhost/webs/SuperOffice.Web.Services` |
+| Name | Description | Default |
+|---|---|---|
+| AllowWebServiceRequests | Permit requests to web services. This value is supported in the default `SoWcfRequestInterceptor` and can be overridden by making a custom `SoWcfRequestInterceptorPlugin`. | true |
+| RemoteBaseURL | Base URL for remote services. Will be overridden by `Services.RemoteBaseURL`. | `http://localhost/webs/SuperOffice.Web.Services` |
```

Comment:
```suggestion
| RemoteBaseURL | Base URL for remote services in multi-tenant environment. Use this setting instead of Services.RemoteBaseURL in CRM Online environments. Will be overridden if `Services.RemoteBaseURL` is set. | `http://localhost/webs/SuperOffice.Web.Services` |
```